src/Shared/Telemetry/TelemetryConstants.ts

```diff
@@ -151,3 +152,8 @@ export enum SourceBlade {
   ScriptExplorer,
   Keys,
 }
+
+export enum Source {
```

Comment:
Make this a string enum. Otherwise I think these will show up in telemetry as integers.